src/model/vectors.ts

```diff
@@ -25,6 +25,10 @@ export class Vector2 {
     return new Vector2(this.x + other.x, this.y + other.y)
   }
 
+  delta(other: Vector2) {
```

Comment:
Feel free to call this 'subtract'. Makes it slightly more general, I suppose

Reply:
Yea, that is a little more general